### PR TITLE
Remove a few more unnecessary packages from the base image

### PR DESCRIPTION
**Pull request by kajinamit**
... to reduce the size of the base image as well as the all images inheriting the base.

### CODE REVIEW

**Comment by raukadah** [comment]
sorry label got added by mistake.

File: container-images/tcib/base/base.yaml [hunk 1]
```diff
@@ -51,12 +51,9 @@ tcib_packages:
   - ca-certificates
   - dumb-init
   - glibc-langpack-en
-  - iscsi-initiator-utils
   - procps-ng
   - python3
-  - rsync
```
[the remark this answers is not in the document]
Comment (kajinamit) [reply container-images/tcib/base/base.yaml:57]: rsync is still required by a few components but we explicitly install the package, so should be installed when really required.

https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/os/swift-base/swift-base.yaml#L12
https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/mariadb/mariadb.yaml#L20

**Review by rabi — APPROVED** [review]

**Pull request merged**